plugins/modules/capabilities.py

```diff
@@ -61,6 +61,12 @@
     path: /bar
     capability: cap_net_bind_service
     state: absent
+
+ - name: Remove all capabilities from /restricted_app
```

Comment:
Please note that your example has a global one-space indentation, which is different from the previous examples (which had no indentation). This results in the `EXAMPLE` block no longer being valid YAML.